### DISCUSSION TITLE
Dokumentation of the ESP Class wanted

### DISCUSSION THREAD

**Discussion by kendo55** [espressif/arduino-esp32#7991] [General]
function like ESP.getChipModel(), ESP.getChipRevision() are still alive

They are defined in:
.....packages\esp32\hardware\esp32\2.0.7\cores\esp32\Esp.cpp

I couldn't find anything in the  [Documentation](https://docs.espressif.com/projects/arduino-esp32/en/latest/)

**Comment by pedrominatel** [comment]
Thanks for the suggestion!
I'll include it in the Docs to-do list.
